keyboards/marksard/leftover30/keymaps/default/rules.mk

```diff
@@ -0,0 +1,3 @@
+RGBLIGHT_ENABLE = yes
+MOUSEKEY_ENABLE = yes
```

Comment:
I'm confused as to why these are defined here.

- Mouse keys is enabled at keyboard level, so setting it here is redundant
- RGB Light is disabled at keyboard level, only to enable it here

Reply:
surely. Change the keyboard level.